README.md

```diff
@@ -110,7 +110,7 @@ The above can then be executed via the `devlab run` command from within the same
 
 ### Multiple Tasks
 
-The example shows a single-command execution configuration, however, Devlab supports named tasks as well. Replace the `task` entry with configuration object `tasks`:
+The example shows a single-command execution configuration, however, Binci supports named tasks as well. Replace the `task` entry with configuration object `tasks`:

```

Comment:
So I _think_ you might be talking about `The above example` and would make to change this to say such? Also,  noticed that example uses `tasks` instead of `task` for it's one liner task you mentioned here for replacing in the next example below.